tests/orquestra/vqa/algorithms/qaoa_test.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pytest
+from orquestra.opt.optimizers import ScipyOptimizer
+from orquestra.quantum.backends.symbolic_simulator import SymbolicSimulator
+from orquestra.quantum.operators import PauliTerm
+
+from orquestra.vqa.algorithms.qaoa import QAOA
+from orquestra.vqa.ansatz.qaoa_farhi import QAOAFarhiAnsatz
+from orquestra.vqa.ansatz.qaoa_warm_start import WarmStartQAOAAnsatz
+from orquestra.vqa.estimation.cvar import CvarEstimator
+
+# def test_if_runs():
+#     hamiltonian = PauliTerm("Z0") + PauliTerm("Z1")
+#     optimizer = ScipyOptimizer(method="L-BFGS-B")
+#     backend = SymbolicSimulator()
+#     qaoa = QAOA.default(cost_hamiltonian=hamiltonian, optimizer=optimizer, n_layers=2)
+#     results = qaoa.find_optimal_params(backend=backend)
+#     breakpoint()
```

Comment:
Remember to remove this before merging :)

Reply:
Changed in https://github.com/zapatacomputing/orquestra-vqa/pull/31/commits/816f8b98ec08a29d896a93911ce41fe65e2058c2